<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="table">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>590</width>
     <height>440</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>130</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Номер договора</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Код абитуриента</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Код формы обучения</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Код факультета</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Код специальности</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Стоимость</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ФИО</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Дата</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_34">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Код факультета</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Append">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_32">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер договора</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_35">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Код специальности</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_33">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Код абитуриента</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Change">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="date">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Код формы обучения</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Стоимость</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ФИО платильщика</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>line</tabstop>
  <tabstop>Append</tabstop>
  <tabstop>table</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
